tasks/sonar_runner.js

```diff
@@ -44,7 +44,10 @@ module.exports = function (grunt) {
         var dryRun = options.dryRun;
         var done = this.async();
 
-        options.sonar.language = options.sonar.language || 'js';
```

Comment:
i think we can have "multi" as the default option. if lang value is multi, options.sonar.language is not set.
that way sonar.language remains documented for readability and consistency.